frontend/src/main.js

```diff
@@ -193,6 +195,8 @@ registerSuspendedVueComponent('task-list', TaskList);
 registerVueComponent('student-list', StudentList);
 registerSuspendedVueComponent('inbus-import', InbusImport);
 registerVueComponent('notifications', NotificationsNew);
+registerSuspendedVueComponent('quiz', Quiz);
```

Comment:
It doesn't look like these components have a top-level `await`, so they don't need to be suspended.